### PR TITLE
DM-42739: Configure Prompt Processing production service for Sasquatch upload

### PR DESCRIPTION
**Pull request by kfindeisen**
This PR allows customization of Prompt Processing's Sasquatch namespace, and configures the production services to upload metrics to a custom namespace.

### CODE REVIEW

File: applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml [hunk 1]
```diff
@@ -45,6 +45,11 @@ prompt-proto-service:
   apdb:
     url: postgresql://rubin@usdf-prompt-processing.slac.stanford.edu:5432/lsst-devl
 
+  sasquatch:
+    endpointUrl: https://usdf-rsp-dev.slac.stanford.edu/sasquatch-rest-proxy
+    namespace: lsst.prompt.prod
```
Comment (isullivan) [inline applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml:50]: Do we want `lsst.prompt.prod` here?

**Review by isullivan — APPROVED** [review]
Please double-check that we want to use `namespace: lsst.prompt.prod` in `values-usdfprod-prompt-processing.yaml` for latiss and lsstcomcamsim, and `namespace: lsst.prompt` everywhere else.

File: applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml (hunk 1 above)
Comment (kfindeisen) [reply applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml:50]: Yes? This is the real LATISS service we run most nights. That's as "production" as we can get ATM.

Comment (kfindeisen) [reply applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml:50]: Resolved in Slack; no specific objections.

**Pull request merged**